indigo/analysis/toc/base.py

```diff
@@ -194,6 +194,10 @@ class TOCBuilderBase(LocaleBasedMatcher):
 
     non_commenceable_toplevel_elements = set(['coverpage', 'preface', 'preamble', 'conclusions', 'attachment', 'component'])
```

Comment:
should `blockContainer` be added to this? Otherwise definitions will show up as commenceable things.